packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -40,6 +40,7 @@ export class FungibleTokenTransferController implements ReactiveController {
   public selectedResource?: Resource;
   public resourceAmount: BigNumber = BigNumber.from(0);
   public destinatonAddress: string = '';
+  public defaultDestinationAddress: string = '';
```

Comment:
Do we need extra property. Why not just update `destinatonAddress` if it's empty?

Reply:
It's true, it will not break any other logic, so we can keep only destinatonAddress